### DISCUSSION TITLE
Install self-hosted Action Runner on bare metal  M1 Mini

### DISCUSSION THREAD

**Discussion by eghaoui** [actions/virtual-environments#6401] [General questions]
Hello,
I would like to install Action Runner on a Self-hosted MacOS M1 mini. I have found the package to be installed on your page: https://github.com/actions/runner-images/blob/releases/macOS-12/20220929/images/macos/macos-12-Readme.md
but i need a guideline to install all these requested applications as i am not gonna use any Virtual environment.

**Comment by al-cheb** [comment]
Currently, we don't support M1 images. You can try to build your own image using Anka(https://veertu.com/download-anka-develop/) with packer template https://github.com/actions/runner-images/blob/main/images/macos/templates/macOS-12.anka.pkr.hcl .

**Reply by eghaoui** [reply]
Thanks for your answer @al-cheb I was trying to create an ansible playbook to deploy the packages on the M1 but it is taking so much time i will try your way.
